packages/gatsby/src/utils/page-ssr-module/bundle-webpack.ts

```diff
@@ -56,6 +58,23 @@ export async function createPageSSRBundle({
   reporter: Reporter
   isVerbose?: boolean
 }): Promise<webpack.Compilation | undefined> {
+  // TODO sc-53539: revisit this transformation
```

Comment:
Can stay a TODO until after merge?

Reply:
This is outdated, I'll remove

Reply:
Done in [`840a313` (#36489)](https://github.com/gatsbyjs/gatsby/pull/36489/commits/840a313564125e0e93ee852eb8015de7981d9edd)